### ISSUE TITLE
[Usability] A11y_Manage.IIS.Net_Microsoft IIS Home Page: "Hide Sidebar" button is not required in the Home Page, since there are no sidebars to show and hide.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#71]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA

**Repro Steps:**
1. Open "Microsoft IIS" Home Page using the URL: https://manage.iis.net/
2. Validate that no any usability or design issue present in the page.

**Actual Result:**
There is "Hide/Show Sidebar" button in the page, but on selecting it, no sidebar appears in the page.

**Expected Result:**
"Hide/Show Sidebar" button is not required in the Home Page, since there are no sidebars to show and hide. So, either it should be disabled, or removed from the page.

**User Impact:**
Since it is an interactive, user will unnecessarily select it and no user action will be performed, since there is no sidebar in the page to show and hide.
Screen reader (visually impaired) users will select it and they may assume that some side bar opened in the page. 

**Attachment:**
[Bug 71_Usability issue with hide sidebar button.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317677/Bug.71_Usability.issue.with.hide.sidebar.button.pptx)

**Comment by jhkimnew** [comment]
By design.  Hide Sidebar is a toggle button and we can't remove the button.
BTW, the button is currently not available for WAC scenario but it should be added back to fix https://github.com/microsoft/IIS.WebManager/issues/391. 
@shhsu Please consider this issue while fixing the issue.
